tests/modules/helloscripting.c

```diff
@@ -28,6 +29,15 @@
  * CONSTI 432    # pushes the value 432 to the top of the stack
  * RETURN        # returns the current value on the top of the stack and marks
  *               # the end of the function declaration.
+ *
+ * FUNCTION sleep # declaration of function 'sleep'
+ * ARGS 0         # pushes the value in the first argument to the top of the
+ *                # stack
+ * SLEEP          # Pops the current value in the stack and sleeps for `value`
```

Comment:
SLEEP is an instruction and `sleep` is the function name. Maybe use another function name to avoid confusion.